ingestion_tools/scripts/importers/annotation.py

```diff
@@ -95,7 +106,15 @@ def __init__(
         if self.file_format not in ["mrc"]:
             raise NotImplementedError("We only support MRC files for segmentation masks")
 
-    def convert(self, fs: FileSystemApi, input_prefix: str, output_prefix: str, voxel_spacing: float):
+    def convert(
+        self,
+        fs: FileSystemApi,
+        input_prefix: str,
+        output_prefix: str,
+        voxel_spacing: float,
+        write_mrc: bool = True,
+        write_zarr: bool = True,
+    ):
         input_file = self.get_source_file(fs, input_prefix)
         return scale_mrcfile(fs, self.get_output_filename(output_prefix), input_file, voxel_spacing=voxel_spacing)
```

Comment:
`write_mrc` and `write_zarr` seem to have no effect because they are not passed to `scale_mrcfile`.

Reply:
Oops! Thanks for catching that! :)